### DISCUSSION TITLE
Achivment Title for old Raid

### DISCUSSION THREAD

**Discussion by Sfaccim** [chromiecraft/chromiecraft#3764] [Suggestions]
I why dont we make challenging old raid like mc and bwl.. why dont we make usfull not only for alt those raid hats now for the main pg are boring. we could implement like a reward for beating the raid with some achivment mount ot even a title like... we beat mc without no one death not even 1 at trash after defeating ragnaros we all receive a title as a reward "The Fire Lord" or insie bwl same challenge but the title would be "The Dragon Slayer" and for Zul Gurub beating Hakkar with all the debuff maybe will unlock the title "The Doctor Plague". for me this would be funny to see some different titles and we be for sure challenging what is not right now for me and many others.

**Comment by Santaria94** [comment]
Titles are strictly limited by the game client, they cannot be changed and new ones added. Also with achievements.

**Comment by Amrx2** [comment]
As mentioned above, to do so would require client modification which is something we are against, so we wouldn't be looking to add this sorry.
